data-analysis/template.yaml

```diff
@@ -137,6 +159,34 @@ Resources:
           OutputLocation:
             !Join ['', ['s3://', !Ref AthenaQueryOutputBucket, '/']]
 
+  ManualAuditDataQueriesWorkgroup:
+    Type: AWS::Athena::WorkGroup
+    Properties:
+      Description: Workgroup for manual queries on TxMA audit data
+      Name: manual-audit-data-queries
+      RecursiveDeleteOption: true
+      WorkGroupConfiguration:
+        EnforceWorkGroupConfiguration: true
+        PublishCloudWatchMetricsEnabled: false
+        ResultConfiguration:
+          EncryptionConfiguration:
+            EncryptionOption: SSE_S3
+          OutputLocation: !Sub s3://${AthenaQueryOutputBucket}/manual-audit-data-queries
```

Comment:
I think the output locations might need a trailing '/'

Reply:
Fixed!